src/main/java/nextstep/fp/Lambda.java

```diff
@@ -27,30 +27,20 @@ public void run() {
     }
 
     public static int sumAll(List<Integer> numbers) {
-        int total = 0;
-        for (int number : numbers) {
-            total += number;
-        }
-        return total;
+        return sumAll(numbers, number -> true);
     }
 
     public static int sumAllEven(List<Integer> numbers) {
-        int total = 0;
-        for (int number : numbers) {
-            if (number % 2 == 0) {
-                total += number;
-            }
-        }
-        return total;
+        return sumAll(numbers, number -> number % 2 == 0);
     }
 
     public static int sumAllOverThree(List<Integer> numbers) {
-        int total = 0;
-        for (int number : numbers) {
-            if (number > 3) {
-                total += number;
-            }
-        }
-        return total;
+        return sumAll(numbers, number -> number > 3);
+    }
+
+    private static int sumAll(List<Integer> numbers, Conditional c) {
+        return numbers.stream()
+                .filter(number -> c.test(number))
```

Comment:
```suggestion
                .filter(c::test)
```
아마 이렇게 표현 가능하지 않을까 생각합니다!